### PR TITLE
graph hw

### PR DESCRIPTION
**Pull request by Leah2356**
Screen Shot 2024-02-28 at 10 13 17 PM

### CODE REVIEW

**Comment by Leah2356** [comment]
My graph is coming up as a bunch of diagonal lines. How do I fix it? Thank you!

File: src/main/java/feiner/projectile/ProjectileGraph.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+public class ProjectileGraph extends JComponent {
+    private Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+        g.translate(0, getHeight());
+        g.setColor(Color.BLACK);
+        int currX = (int) projectile.getX();
```
Comment (andrewoid) [inline src/main/java/feiner/projectile/ProjectileGraph.java]: add projectile.setSeconds(0) before you getX() here.

File: src/main/java/feiner/projectile/ProjectileGraph.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        projectile.setSeconds(0);
+
+        for (int i = 0; i <= projectile.getApexTime() * 2 + 1; i++) {
+            int currX = (int) projectile.getX();
+            int currY = (int) projectile.getY();
+            projectile.setSeconds(i);
+            g.drawLine(currX, -currY, (int) projectile.getX(), -(int) projectile.getY());
+        }
+        g.setColor(Color.BLUE);
+        g.fillOval((int) projectile.getInterceptX() / 2, (int) -projectile.getPeakY(), 8, 8);
```
Comment (andrewoid) [inline src/main/java/feiner/projectile/ProjectileGraph.java]: center the circle on the curge.

**Comment by andrewoid** [comment]
Add a screenshot

**Review by andrewoid — APPROVED** [review]

**Pull request merged**